about.html

```diff
@@ -0,0 +1,124 @@
+                    <h1 class="about-description">Annual Art And Literary Appreciation 2022!</h1>
+                </div>
+                <div id="about-texts"><br>The Annual Art And Literary Appreciation brings together a community of
+                    experts, academics, and musicians who comprise the AALA affiliate
+                    network across the globe. <br>This year we hope to reach a new milestone where our experts offer
+                    mentorship tofour lucky mentees</br><br>so, this year, if you hope and are open to achieving
+                    something really huge in literary Appreciation, music, arts, culture, and history this summit is for
+                    you -</br>
+                    <br>We hope you'll consider joining us this year to build experience and learn from the best whilst
+                    you foster your own growth
+                    </p>
+                </div>
+               <div id="contacts"><p class="contact">Please contact us via Email for any further questions about AALA 2022</p></div> 
+                <div id="contact">Hardejuwon@gmail.com</div>
+            </div>
+        </div>
+    </section>
+
+
+     <article class="sectioned2">
+        <div id="sectioned2">
+            <div id="AALA">
+                <h1 class="AALA1">Annual Art And Literary Appreciation 2022 logo</h1>
+            </div>
+            <div class="AALA-text">
+                <p id="AALA-text">The logo of AALA 2022 was decided together with the partners who comprised the first
+                    ever made summit in 2015. <br>
+                    After an online public voting, the design made by Fatai Fadipe was chosen. </br><br>After
+                    consultation with Mr. fadipe we have optimized the logo for this years AALA</br>
+                </p>
+            </div>
+            <div class="desktopheader">AALA 2022</div>
+        </div>
+    </article> 
+
+    <article class="sectioned33">
+        <div class="sectioned3">
+            <h2 class="timewarped">See The Past AALA Global Summits</h2>
+            <p id="timewarp">Take a look at the AALA summits which took place in Belarus and Nigeria.</p>
+            <div class="timewarpimage">
+                <div class="aboutimg1">
+                    <div class="textcontz">
+                        <div class="img1txt">2018</div>
+                        <div id="img1txt">AALA Global Summit in belarus</div>
+                    </div>
+                </div>
+                <div class="aboutimg2">
+                    <div class="textcontz">
+                        <div class="imgtxt2">2015</div>
+                        <div class="imgtxt2">AALA Global Summit in, Nigeria</div>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        </div>
+    </article>
+
+    <footer class='footsie'>
+        <div class="bou-footer">
+            <div class="boufootertext">AALA</div>
```

Comment:
It's better to make the `text-color` of the logo white so that it's visible 🤗

![image](https://user-images.githubusercontent.com/73714615/190649746-afe94105-3fcb-4520-b87f-4fb7d378a9b4.png)

Reply:
done
thanks